### DISCUSSION TITLE
Starter or example

### DISCUSSION THREAD

**Discussion by rafcontreras** [sergiodxa/remix-hono#16] [Q&A] [answered]
Do you have a starter or example to see how to implement this?

**Comment by sergiodxa** [comment, marked as answer]
Right now you can check [my website](https://github.com/sergiodxa/sergiodxa.com/blob/main/server/index.ts) for an implementation. As I test this more there and on a closed source app I'm using it I will create a basic starter/Remix stack to initialize a new app.

**Reply by eugene-rn** [reply]
Noticed that you removed Hono from the project. It would be interesting to know why and whether it is worth waiting for the starter kit with Hono?

**Comment by dusty** [comment]
@rafcontreras and @sergiodxa 

Been playing around with this and I have what seems to be a working example using this with node.

https://github.com/dusty/remix-hono-experiment
